.github/workflows/release-please.yml

```diff
@@ -0,0 +1,19 @@
+on:
+  push:
```

Comment:
It should be on_release I guess, not on_push 100%

Reply:
I believe it should be `on_push` @SemyonSinchenko because on a push to main (when you are ready for a release), this will create another PR listing all the changes that has gone through to the `main` since the previous release.

@dinjazelena - yes I have tested this on this repo https://github.com/nijanthanvijayakumar/release-playground. The version at least for `Quinn` is being tracked under the `.release-please-manifest.json` file.